test/test_nexus.py

```diff
@@ -297,6 +305,20 @@ def test_closestore(caplog):
     assert True
 
 
+def test_falsly_delete_store(caplog):
```

Comment:
Should be `test_store_already_deleted_issues_warning`. I think I'm okay with this being a warning and not an error.